pkg/session/container.go

```diff
@@ -113,3 +113,27 @@ func (x *ContainerContext) ForSetEACL() {
 func (x *ContainerContext) IsForSetEACL() bool {
 	return x.isForVerb(session.ContainerVerbSetEACL)
 }
+
+// Marshal marshals ContainerContext into a protobuf binary form.
+func (x *ContainerContext) Marshal(bs ...[]byte) ([]byte, error) {
+	var buf []byte
+	if len(bs) > 0 {
+		buf = bs[0]
+	}
+	return x.ToV2().StableMarshal(buf)
+}
+
+// Unmarshal unmarshals protobuf binary representation of ContainerContext
```

Comment:
```suggestion
// Unmarshal unmarshals protobuf binary representation of ContainerContext.
```

Reply:
Sorry, typo. And linter said nothing about it.
Fixed.